pom.xml

```diff
@@ -221,22 +221,6 @@ limitations under the License.
       </plugins>
     </pluginManagement>
     <plugins>
-      <plugin>
```

Comment:
unrelated to this PR but removed to reduce noise - we have a separate lint check now, don't need to include this in the main build path